recipes/pystackreg/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pystackreg" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "4ab1c2cee5233a96faf29de47b8ace3892270f2334a7a3cb49e3369fb2db8fec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy
+    - pip
+  run:
+    - python
+    - numpy
+    - tqdm
+
+test:
+  imports:
+    - pystackreg
+
+about:
+  home: https://bitbucket.org/glichtner/pystackreg
+  license: Custom
+  license_family: OTHER
+  license_file: LICENSE.txt
+  summary: Python implementation of the ImageJ/FIJI Plugin TurboReg/StackReg
```

You can include the citation in the README and Anaconda.org page using something like
```suggestion
  summary: Python implementation of the ImageJ/FIJI Plugin TurboReg/StackReg
  description: |
    Some further description...
 
    Based in the work in X.
```

---

Added. See below.